composer.json

```diff
@@ -8,14 +8,23 @@
   "minimum-stability": "stable",
   "autoload": {
     "psr-4": {
-      "": "src/",
+      "Google\\Photos\\Library\\V1\\": "src/Google/Photos/Library/V1/",
+      "GPBMetadata\\Google\\Photos\\Library\\V1\\": "src/GPBMetadata/Google/Photos/Library/V1/"
```

Comment:
Let's shorten these to `Google\\Photos\\` and `GPBMetadata\Google\Photos\` - this way the `Types` folders are included as well.

Reply:
took the slightly lazier option of shortening them to `"Google\\"` and `"GPBMetadata\\"`.